Description: Fix endian issue with lcms2 i/face
 In the lcms2 interface code, an endian flag was being set on big endian
 platforms, and it shouldn't be.
Origin: http://git.ghostscript.com/?p=ghostpdl.git;a=commit;h=67a924e
Author: Chris Liddell <Chris.Liddell@artifex.com>
Forwarded: yes
Bug: http://bugs.ghostscript.com/show_bug.cgi?id=692962
Last-Update: 2012-04-27

--- a/base/gsicc_lcms2.c
+++ b/base/gsicc_lcms2.c
@@ -329,7 +329,7 @@
       when we use the transformation. */
     src_data_type = (COLORSPACE_SH(lcms_src_color_space)|
                         CHANNELS_SH(src_nChannels)|BYTES_SH(2));
-#if arch_is_big_endian
+#if 0
     src_data_type = src_data_type | ENDIAN16_SH(1);
 #endif
     if (lcms_deshandle != NULL) {
@@ -344,7 +344,7 @@
     des_data_type = (COLORSPACE_SH(lcms_des_color_space)|
                         CHANNELS_SH(des_nChannels)|BYTES_SH(2));
     /* endian */
-#if arch_is_big_endian
+#if 0
     des_data_type = des_data_type | ENDIAN16_SH(1);
 #endif
 /* Create the link */
